### ISSUE TITLE
When i use Proxifier,Dbeaver cannot get SOCK

### ISSUE THREAD

**Issue by lxkplus** [dbeaver/dbeaver#11147]
sorry for my poor English.
I konw dbeaver support SOCK,but when i use Proxifier to get socks,Dbeaver cannot get socks.
 Dbeaver incompatible with Proxifier，
let me put it another way：if i set a global socks,dbeaver cannot get zhe socks 
it is a small probrem，but i think i should tell others .

and when i import sql which its decode is gbk，unicode or others，i cannot read it corectly。
when i use navicat，it can Automatic transcoding。
There is no button for transcoding，so i  can open sql by .txt and CRTL +C,thenCRTL +V.

that's all.

**Comment by kseniiaguzeeva** [comment]
1.  How do you set proxy in DBeaver?
In connection settings
![dbeaver_q76j38DNye](https://user-images.githubusercontent.com/51405061/107608446-12e1c680-6c4d-11eb-9417-91c2486ab585.png)

or in global settings
![dbeaver_X5SepZipGb](https://user-images.githubusercontent.com/51405061/107608456-1b3a0180-6c4d-11eb-8363-737d25b4d1f1.png)

2. Could you create another ticket for the second issue and give an SQL file to reproduce?

**Comment by lxkplus** [comment]
Thank you for your reply.
This problem has been solved, I have set up SSH, now there is no problem using it.
But I think the strange thing is that when I use proxifer, dbeaver can't proxy through proxifer.
When using agent, dbeaver always seems to say: I have my own agent system. You look down on me when using this kind of software. Then he won't go.

I don't know if this is a bug.
Thank you again for your reply.
